order/parse_trade_clear.go

```diff
@@ -0,0 +1,104 @@
+package order
+
+import (
+	"github.com/huobirdcenter/huobi_golang/pkg/model/order"
+	"math"
+)
+
+func (OrderDataClient *OrderDataClient) ParseTradeClearing(response order.SubscribeTradeClearResponse) {
```

Comment:
```suggestion
func (c *OrderDataClient) ParseTradeClearing(response order.SubscribeTradeClearResponse) {
```

Not sure why you wrote the two methods in separate files but it's fine to write them into a single file.

Reply:
a good practice would be writing a `interface.go` file that contains the following code:
```
type Interface interface {
ParseOrder(...) ...
ParseTradeClearing(...) ...
}
```

then in a `service.go` file you implement the two methods. However, it's highly recommended that you don't expose the `OrderDataClient`. I'll try to make an example here for your reference...however, I do know that the `OrderDataClient` may need to be shared in multiple packages.

Reply:
> a good practice would be writing a `interface.go` file that contains the following code:
> 
> ```
> type Interface interface {
> ParseOrder(...) ...
> ParseTradeClearing(...) ...
> }
> ```
> 
> then in a `service.go` file you implement the two methods. However, it's highly recommended that you don't expose the `OrderDataClient`. I'll try to make an example here for your reference...however, I do know that the `OrderDataClient` may need to be shared in multiple packages.

get it